tests/blob/apis/blockblob.test.ts

```diff
@@ -340,6 +340,23 @@ describe("BlockBlobAPIs", () => {
     );
   });
 
+  it("commitBlockList should return versionId as undefined @loki @sql", async () => {
```

Comment:
should add test case to 
1. validate put blob create blob version for block/page blob. 
2. Commit blob also create blob version.